apps/rush-lib/src/cli/logic/npm/NpmShrinkwrapFile.ts

```diff
@@ -45,6 +45,10 @@ export class NpmShrinkwrapFile extends BaseShrinkwrapFile {
     return this._getTempProjectNames(this._shrinkwrapJson.dependencies);
   }
 
+  protected serialize(): string {
+    return JSON.stringify(this._shrinkwrapJson, undefined, 2);
```

Comment:
>return JSON.stringify(this._shrinkwrapJson, undefined, 2); [](start = 4, length = 58)

Is this guaranteed not to alter the NPM shrinkwrap file?

Reply:
I believe so

---
In reply to: [170406457](https://github.com/Microsoft/web-build-tools/pull/543#discussion_r170406457) [](ancestors = 170406457)